haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
I am pretty sure it is (except perhaps on Windows, which is always different, but that is perhaps not a worry?)